testdata/p4tc_samples_outputs/digest.template

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/digest numtables 1
+
+$TC p4template create action/digest/ingress/send_nh actid 1 \
+	param port type dev \
+	param srcMac type macaddr \
+	param dstMac type macaddr
+$TC p4template update action/digest/ingress/send_nh state active
+
+$TC p4template create action/digest/ingress/drop actid 2
+$TC p4template update action/digest/ingress/drop state active
+
+$TC p4template create extern/root/Digest extid 0x05000000 numinstances 1 tc_acl 0x19b6 has_exec_method
```

Comment:
Hmm, maybe this is an issue
We have `digest` in lower case in the examples and doc, but it's called `Digest` in P4
I think that's why the compiler is generating it capitalising the first letter

If possible, it would help if the compiler could generate it in lower case
```suggestion
$TC p4template create extern/root/digest extid 0x05000000 numinstances 1 tc_acl 0x19b6 has_exec_method
```

If not, we should discuss further

Reply:
I think we should follow the same format as P4 Program to keep consistency.

Reply:
Okk